components/callbacks.py

```diff
@@ -446,3 +449,38 @@ async def buy(update: Update, context: ContextTypes.DEFAULT_TYPE) -> None:
         await try_to_delete(message.reply_to_message)
 
     await try_to_delete(message)
+
+
+async def _token_warning(
+    update: Update, context: ContextTypes.DEFAULT_TYPE, middle_text: str = ""
+) -> None:
+    """Warn people when they share their bot's token, and tell them to revoke it"""
+    message = cast(Message, update.effective_message)
+
+    # Update timestamp on chat_data, and get "x time since last time" text
+    last_time = update_shared_token_timestamp(update, context)
+
+    # Send the message
+    await message.reply_text(f"{TOKEN_TEXT}{middle_text}Previous token was shared: {last_time}")
```

Comment:
```suggestion
    await message.reply_text(f"{TOKEN_TEXT}{middle_text}Days since last token was shared: {last_time}")
```

just a phrasing suggestion to better align with the popular meme :D